### ISSUE TITLE
Add ability to edit/rename entries

### ISSUE THREAD

**Issue by idelsink** [Place1/wg-access-server#100]
I would be nice if we can rename/change the name of a device to something else.

For example via an edit button in the overflow menu:

![image](https://user-images.githubusercontent.com/8396902/105640878-f0942e80-5e80-11eb-8ef5-48a77ed6752a.png)

**Comment by talesam** [comment]
I agree with you, having to edit any changes is much simpler than deleting a connection, creating another one and configuring it again on the client, even more if it is a TV, for example.
